app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -34,26 +34,68 @@ tm-page(:title='validatorTitle(this.validator)')
       tm-list-item(dt="Commission Maximum" :dd="pretty(validator.commission_max) + ' %'")
       tm-list-item(dt="Commission Change-Rate" :dd="pretty(validator.commission_change_rate) + ' %'")
       tm-list-item(dt="Commission Change Today" :dd="pretty(validator.commission_change_today) + ' %'")
+
+    modal-stake(
+      v-if="showModalStake"
+      v-on:submitDelegation="submitDelegation"
+      :showModalStake.sync="showModalStake"
+      :fromOptions="[{ key: `My Wallet - ${this.wallet.address}`, value: 0 }]"
+      :maximum="availableAtoms()"
+      :to="validator.owner"
+    )
+
+    tm-modal(:close="closeCannotStake" icon="warning" v-if="showCannotStake")
```

Comment:
Are you sure, we want this to be a modal? I thought it is discouraged to use blocking modals in modern UX. (Not a blocker, we can have follow up on this)

Reply:
yeah, this should be a notification